.github/workflows/espresso.yml

```diff
@@ -14,41 +11,38 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
-      matrix:
-        python-version:
-        - "3.12"
 
     steps:
       - uses: actions/checkout@v4
-      - uses: mpi4py/setup-mpi@v1
-      - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v5
+      - name: Setup EESSI
+        uses: eessi/github-action-eessi@v3
         with:
-          python-version: ${{ matrix.python-version }}
+          eessi_stack_version: "2023.06"
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip
-          python -m pip install -r dev-requirements.txt
-      - name: Install package
-        run: |
-          python -m pip install ".[rnd]"
-      - name: Install ESPResSo
-        run: |
-          sudo apt-get install libboost-all-dev
-          git clone -b python --recursive https://github.com/espressomd/espresso.git
-          cd espresso
-          python -m pip install -r requirements.txt
-          mkdir build && cd build
-          cp ../maintainer/configs/maxset.hpp myconfig.hpp
-          cmake ../ -D ESPRESSO_BUILD_WITH_WALBERLA=on -D ESPRESSO_BUILD_WITH_FFTW=OFF
-          cmake --build . -j $(nproc)
+          test "${RUNNER_ARCH}" = "X64" && module use /cvmfs/dev.eessi.io/espresso/versions/2023.06/software/linux/x86_64/amd/zen2/modules/all
+          module load ESPResSo/dc87ede3f6c218bb71624460752bc8c26a271c33-foss-2023b
```

Comment:
update: with https://github.com/easybuilders/easybuild-easyconfigs/pull/24421 ESPResSo can be built for foss/2025a:
```
{EESSI 2025.06} jgrad@lama:~$ module load ESPResSo/dc87ede-foss-2025a
{EESSI 2025.06} jgrad@lama:~$ python
Python 3.13.1 (main, Oct 22 2025, 21:38:59) [GCC 14.2.0] on linux
>>> import numpy, espressomd
>>> print(f"numpy {numpy.__version__}, ESPResSo {espressomd.__version__}")
numpy 2.3.1, ESPResSo 5.0.0
```